src/tests/infusion/deleteInfusion.test.js

```diff
@@ -0,0 +1,52 @@
+import supertest from 'supertest';
+import app from '../../http/app';
+import db from '../../db';
+import testRunner from '../utils/testRunner';
+
+const { WARD_USER } = db.users.userTypes;
+
+const request = supertest(app);
+
+const infusion = {
+  startVolume: 700,
+  stopVolume: 50,
+  patientName: 'Tumtum',
+  doctorsInstruction: 'This is the doctor\'s instructions and it\'s a string',
+  deviceId: '5db95971c9da2412401b1804',
+};
+
+const testCases = [
+  {
+    title: 'should delete infusion',
+    request: context => ({
+      body: {},
+      path: `/api/infusion/${context.infusionId}`,
+      method: 'delete',
+      headers: {
+        'req-token': context.testGlobals[WARD_USER].authToken,
+      },
+    }),
+    response: {
+      status: 204,
+      body: {},
+    },
+  },
+];
```

Comment:
How about other test cases such as if the user is not providing authentication creds, if the id of infusion to delete is not valid e.tc. We need tests for those.

Reply:
fixed